ports/esp32/main/CMakeLists.txt

```diff
@@ -0,0 +1,114 @@
+# Set location of base MicroPython directory, and this port's directory.
```

Comment:
Does this file have to be in a subdirectory? (I know nothing about cmake)

Reply:
Yes.  Or at least I couldn't work out how to put it in `ports/esp32/` directly...